packages/clerk-js/src/core/clerk.ts

```diff
@@ -296,7 +301,7 @@ export default class Clerk implements ClerkInterface {
     //1. setLastActiveSession to passed usersession (add a param).
     //   Note that this will also update the session's active organization
     //   id.
-    if (session && typeof document != 'undefined' && document.hasFocus()) {
+    if (typeof document != 'undefined' && document.hasFocus()) {
```

Comment:
❓ 
Why was the check removed here ?

Reply:
Scratch it, saw `#touchLastActiveSession` handles falsy values.

Reply:
Yep, I actually removed these checks to align it with the `setSession` implementation. The only difference is basically that in the `setActive` the session can `undefined`. But if it is, we use the current session, so at this point we have the same state as `setSession`, so no need for the extra check.